custom/ewsghana/templates/ewsghana/multi_report.html

```diff
@@ -4,6 +4,9 @@
 {% load i18n %}
 
 {% block reportcontent %}
+    {% if report.slug|safe == 'stock_summary_report' or report.slug|safe == 'cms_rms_summary_report' %}
```

Comment:
feels like this type of hard-coding could be a bit brittle. maybe you could make an attribute on the report class like `show_title` or something and pass it to the template?